src/invoices/InvoicesList/Invoice/utils.js

```diff
@@ -35,3 +35,28 @@ export const createInvoiceLineFromPOL = (poLine, invoiceId, vendor) => {
     ...optionalProps,
   };
 };
+
+export const getAlwaysShownAdjustmentsList = (configs) => {
+  const shownAdjustmentslist = configs.map(({ id, value = {} }) => {
```

Comment:
it looks like a copy-paste of `getSettingsAdjustmentsList`

Reply:
additionally there is functionality to get adjustment from config, it can be reused somehow